code/modules/power/cells/power_cells.dm

```diff
@@ -1,137 +1,92 @@
-
-/obj/item/weapon/cell/hyper/empty/New()
-	..()
-	charge = 0
-
-/obj/item/weapon/cell/infinite
-	name = "infinite-capacity power cell!"
-	icon_state = "icell"
-	origin_tech =  null
-	maxcharge = 30000 //determines how badly mobs get shocked
-	matter = list(DEFAULT_WALL_MATERIAL = 700, "glass" = 80)
-
-	check_charge()
-		return 1
-	use()
-		return 1
-
-/obj/item/weapon/cell/potato
-	name = "potato battery"
-	desc = "A rechargable starch based power cell."
-	origin_tech = list(TECH_POWER = 1)
-	icon = 'icons/obj/power.dmi' //'icons/obj/harvest.dmi'
-	icon_state = "potato_cell" //"potato_battery"
-	charge = 100
-	maxcharge = 300
-	minor_fault = 1
-
-
-/obj/item/weapon/cell/slime
-	name = "charged slime core"
-	desc = "A yellow slime core infused with phoron, it crackles with power."
-	origin_tech = list(TECH_POWER = 4, TECH_BIO = 5)
-	icon = 'icons/mob/slimes.dmi' //'icons/obj/harvest.dmi'
-	icon_state = "yellow slime extract" //"potato_battery"
-	description_info = "This 'cell' holds a max charge of 10k and self recharges over time."
-	maxcharge = 10000
-	matter = null
-	self_recharge = TRUE
+/obj/item/weapon/cell/crap
+	name = "\improper rechargable AA battery"
+	desc = "You can't top the plasma top." //TOTALLY TRADEMARK INFRINGEMENT
+	origin_tech = list(TECH_POWER = 0)
+	maxcharge = 500
+	matter = list(DEFAULT_WALL_MATERIAL = 700, "glass" = 40)
+
+/obj/item/weapon/cell/crap/empty/New()
+	..()
+	charge = 0
+
+/obj/item/weapon/cell/secborg
+	name = "security borg rechargable D battery"
+	origin_tech = list(TECH_POWER = 0)
+	maxcharge = 600	//600 max charge / 100 charge per shot = six shots
+	matter = list(DEFAULT_WALL_MATERIAL = 700, "glass" = 40)
+
+/obj/item/weapon/cell/secborg/empty/New()
+	..()
+	charge = 0
+
+/obj/item/weapon/cell/apc
+	name = "heavy-duty power cell"
+	origin_tech = list(TECH_POWER = 1)
+	maxcharge = 5000
+	matter = list(DEFAULT_WALL_MATERIAL = 700, "glass" = 50)
+
+/obj/item/weapon/cell/high
+	name = "high-capacity power cell"
+	origin_tech = list(TECH_POWER = 2)
+	icon_state = "hcell"
+	maxcharge = 10000
+	matter = list(DEFAULT_WALL_MATERIAL = 700, "glass" = 60)
+
+/obj/item/weapon/cell/high/empty/New()
+	..()
+	charge = 0
+
+/obj/item/weapon/cell/super
+	name = "super-capacity power cell"
+	origin_tech = list(TECH_POWER = 5)
+	icon_state = "scell"
+	maxcharge = 20000
+	matter = list(DEFAULT_WALL_MATERIAL = 700, "glass" = 70)
+
+/obj/item/weapon/cell/super/empty/New()
+	..()
+	charge = 0
+
+/obj/item/weapon/cell/hyper
+	name = "hyper-capacity power cell"
+	origin_tech = list(TECH_POWER = 6)
+	icon_state = "hpcell"
+	maxcharge = 30000
+	matter = list(DEFAULT_WALL_MATERIAL = 700, "glass" = 80)
+
+/obj/item/weapon/cell/hyper/empty/New()
+	..()
+	charge = 0
+
+/obj/item/weapon/cell/infinite
+	name = "infinite-capacity power cell!"
+	icon_state = "icell"
+	origin_tech =  null
+	maxcharge = 30000 //determines how badly mobs get shocked
+	matter = list(DEFAULT_WALL_MATERIAL = 700, "glass" = 80)
+
+	check_charge()
```

Comment:
Not swapped to absolute pathing?

Reply:
Ah, good catch, that was a straight copy-paste from the old file.